Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -405,6 +405,17 @@ internal static void Update()
             if (!active)
                 return;
 
+            // Check for dead clients but exclude the host client because it
+            // doesn't ping itself and therefore may appear inactive.
+            foreach (NetworkConnectionToClient conn in connections.Values)
```

Comment:
did you do any profiling on this one?
if we have 1k connections / players, then Update is going to iterate 1k connections every single time now.

is there any more performant way?
what if we check the timeout in connection.send? we call this anyway

Reply:
well actually it's gonna be hard to test this with 1k connections.
but this might be a significant cost nonetheless. 
I almost think we need to run a 500 CCU benchmark with and without it.
otherwise we might end up with a bad surprise at some point

Reply:
added bool disconnectInactiveConnections

Reply:
resolved in discord